ci/values.yaml

```diff
@@ -65,6 +65,28 @@ config:
           enabled: true
         container.uptime:
           enabled: true
+        container.blockio.io_service_bytes_recursive:
+          enabled: true
+        container.cpu.utilization:
+          enabled: true
+        container.cpu.percent:
+          enabled: true
+        container.memory.hierarchical_memory_limit:
+          enabled: true
+        container.memory.active_anon:
+          enabled: true
+        container.network.io.usage.tx_bytes:
```

Comment:
So weird, these seem to be enabled by default based on https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/metadata.yaml